float64 imu0
float64 imu1
float64 imu2
float64 rc0
float64 rc1
float64 rc2
float64 rel0
float64 rel1
float64 rel2
float64 imuvel0
float64 imuvel1
float64 imuvel2
float64 relvel0
float64 relvel1
float64 relvel2
